### ISSUE TITLE
Add option for different server render distance to client?

### ISSUE THREAD

**Issue by Ashley-Cause** [Johni0702/bobby#330]
When I'm playing on servers, most of which are located far away from me in the US or someplace else, I always get massive lag spikes loading chunks.

Is there a way to set a different render distance for the server, or such that the client only requests a smaller render distance before using fake chunks?
E.g. With a server render distance of 6 chunks (as is I think(?) the default on most servers) changing a setting so that the server thinks my render distance is 3 or something, and then bobby defaults to fake chunks for the other 14 chunks or something if I had 16 chunks render.

TLDR: New option for different max render distance, e.g. 16 client render, 2 server, so that only render of 2 is sent to server, remaining 14 are fake chunks.

**Comment by Johni0702** [comment]
No, the server is in control of sending chunks to the client.
And while the client does send its render distance to the server, I've yet to see a server which actually respects that (if you find one, then I may consider adding an option for this, but without the server cooperating, such an option would be useless).
